src/utils.ts

```diff
@@ -7,3 +7,32 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function handlePageChange(
+  page: number,
+  setCurrentPage: (page: number) => void,
+) {
+  setCurrentPage(page);
+}
+
+export function calculateItems(
+  currentPage: number,
+  perPage: number,
+  totalItems: number,
+): [number, number] {
+  const startItem = (currentPage - 1) * perPage + 1;
+  const endItem = Math.min(currentPage * perPage, totalItems);
+
+  return [startItem, endItem];
+}
+
+export function handlePerPageChange(
+  e: React.ChangeEvent<HTMLSelectElement>,
+  setPerPage: (perPage: number) => void,
+  setCurrentPage: (page: number) => void,
+) {
+  const newPerPage = parseInt(e.target.value, 10);
+
+  setPerPage(newPerPage);
+  setCurrentPage(1);
+}
```

Comment:
it's redundant to move function handlers to separate files, better to keep them in the component